modin/pandas/dataframe.py

```diff
@@ -1525,7 +1525,22 @@ def mul(
             broadcast=isinstance(other, Series),
         )
 
-    rmul = multiply = mul
+    multiply = mul
+
+    def rmul(
+        self, other, axis="columns", level=None, fill_value=None
+    ):  # noqa: PR01, RT01, D200
+        """
+        Get multiplication of ``DataFrame`` and `other`, element-wise (binary operator `mul`).
```

Comment:
```suggestion
        Get multiplication of `other` and ``DataFrame``, element-wise (binary operator `mul`).
```

Reply:
I agree that what you suggest is clearer, but I would like to keep this as is to match the pandas API documentation.